armadillo/src/main/java/org/molgenis/armadillo/ArmadilloSession.java

```diff
@@ -61,20 +66,31 @@ void tryDetachRConnection(RConnection connection) {
   RConnection getRConnection() {
     try {
       if (rSession == null) {
-        return connectionFactory.createConnection();
+        var connection = connectionFactory.createConnection();
+        pid = processService.getPid(connection);
+        return connection;
```

Comment:
Maybe log the pid as well? We ports, but pid's would be useful as well concerning you can execute a api call to kill it?
Or do we use the connection to determine the pid using the actuator endpoints?